components/org.wso2.carbon.identity.api.server.authenticators/org.wso2.carbon.identity.api.server.authenticators.v1/src/main/resources/authenticators.yaml

```diff
@@ -387,13 +504,17 @@ components:
         description:
           type: string
           example: The user defined custom local authenticator.
+        amrValue:
+          type: string
+          example: basic
         endpoint:
           $ref: '#/components/schemas/Endpoint'
       required:
         - name
         - displayName
         - isEnabled
         - endpoint
+        - amrValue
```

Comment:
remove this; as this should be considered as an optional parameter. Ideally we can use the authenticator name as the amr value in case this is null.